src/entities/tasks/tasksAdapter.ts

```diff
@@ -7,4 +7,10 @@ export const tasksAdapter = createEntityAdapter<Task>({
   sortComparer: (a, b) => a.id - b.id,
 });
 
-export const taskSelectors = tasksAdapter.getSelectors((state: RootState) => state.core.entities.tasks);
+export const adaptorSelectors = tasksAdapter.getSelectors((state: RootState) => state.core.entities.tasks);
+
+export const selectNotStartableNotDone: (state: RootState) => Task[] = createSelector(adaptorSelectors.selectAll, (tasks: Task[]) =>
+  tasks.filter((task) => task.is_done === false && task.is_startable === false)
```

Comment:
Can be shortened a bit, i think
```suggestion
  tasks.filter((task) => !task.is_done && !task.is_startable)
```